README.md

```diff
@@ -31,7 +33,7 @@ admission control can be delegated to controllers such as [gatekeeper](https://g
 - ✔️ Up-to-date [documentation](/docs).
```

Comment:
How do you get the rough test coverage rate 72%? The unit test is about 66% with `go tool cover -func=$(ARTIFACTS)/cover.out`.

Reply:
I remember adding together the coverage of unit and integration. The reason why unit coverage is somewhat low is because the controllers are primarily tested with integration tests.

Reply:
Separate the unit tests and integration tests apart.